pyvo/registry/regtap.py

```diff
@@ -128,6 +131,12 @@ def search(*constraints:rtcons.Constraint, includeaux=False, **kwargs):
         This may result in duplicate capabilities being returned,
         especially if the servicetype is not specified.
 
+    maxrec : int
+        Overrides the RegTAP server's default limit on the number of rows to
+        return.  You may need to use this if you want to retrieve more
+        than a few thousand matches.  Note that truncated search results
+        are not reproducible.
```

Comment:
```suggestion
        Overrides the RegTAP server's default limit on the number of rows to
        return.  You may need to use this if you want to retrieve more
        than a few thousand matches. 
        The server may also have a hard limit that ``maxrec`` cannot override.  
        Please note that truncated search results are not reproducible.
```

Reply:
My reading of the spec is that the server has the last word and might override the requested `MAXREC` as I highlighted in my comment. The rest looks OK so I'll also approve it to fix the doctest.